src/toggle/GenreToggle.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './genreToggle.css';
+
+function GenreToggle() {
+    const [toggle, setToggle] = React.useState(false);
+
+    return (
+        <div className='genre_toggle__wrapper'>
+           <button className='genre_toggle__button' onClick={() => setToggle(e => !e)}>Select Genre</button>
+
+            {toggle ? <div className='genre_toggle__options'>
```

Comment:
Instead of a ternary operator, it would be nice to do something like this:
```suggestion
            {toggle && <div className='genre_toggle__options'>
```